src/anet.c

```diff
@@ -151,7 +151,7 @@ int anetCloexec(int fd) {
 
 /* Enable TCP keep-alive mechanism to detect dead peers,
  * TCP_KEEPIDLE, TCP_KEEPINTVL and TCP_KEEPCNT will be set accordingly. */
-int anetKeepAlive(char *err, int fd, int interval) {
+int anetKeepAlive(char *err, int fd, int interval, const int probes) {
```

Comment:
`const` isn’t commonly used for by-value parameters in Valkey. For consistency with the codebase, I'd remove it.

Reply:
OK.
[2fef5e8](https://github.com/valkey-io/valkey/pull/1765/commits/2fef5e8f4d3218c51b4082e40164ac2085f3f8e0)